### DISCUSSION TITLE
New to Github

### DISCUSSION THREAD

**Discussion by Aimpact-naveen** [community/community#125698] [New to GitHub] [answered]
Hi, my name is Naveen. I am based in Europe and trying to dive into developing web applications using Microsoft. Also new to Github. It is bit overwhelming coming across so many terminologies. Any suggestions welcome!

**Comment by Carlo-MG** [comment]
Hola, @Aimpact-naveen.

Bienvenido a GitHub. Al igual que tú, yo también soy nuevo en la plataforma.

Las únicas recomendaciones que puedo proporcionarte son:

Aprender algo nuevo cada día: No me refiero a aprender una nueva tecnología cada día, sino a enfocarte en aprender una nueva funcionalidad en el lenguaje que estés estudiando. Esto te ayudará a dominarlo con el tiempo.

Enfocarse en una tecnología a la vez: No intentes abarcar todas las tecnologías de la noche a la mañana. Es mejor aprender una a la vez y de manera constante. Por ejemplo, yo estoy aprendiendo Flask para la creación de APIs.

Espero que estas sugerencias te sean útiles. ¡Buena suerte!

**Reply by Aimpact-naveen** [reply]
Hi Carlo, thanks for your welcome message and a indeed very good suggestion. I absolutely agree. 

I was trying to take a course on Github, was bit too much. How anyone else who is solo newcomer (like me) to Github, I would strongly recommend this video https://www.youtube.com/watch?v=Oaj3RBIoGFc
So, after watching this earlier today, I am "good" with github. 

Now, I want to create my first (Very basic) web app. I want to stay Azure native as much as possible. the course from azure "Developing your first web application with React" is bit too much ! If anyone can suggest a simple tutorial to start with, will appreciate. 

Regards, Naveen

**Reply by Carlo-MG** [reply, marked as answer]
Hola Naveen,

Si quieres crear tu primera aplicación web, no te recomendaría empezar con React a menos que ya tengas una buena base en JavaScript. Si aún no la tienes, te sugiero que comiences con proyectos pequeños para mejorar tus fundamentos en JavaScript y HTML. Una vez que te sientas cómodo con estos, será más fácil aprender y trabajar con React.

En cuanto a los tutoriales, no intentes ver un tutorial de unas 4 horas en un solo día. Es mejor dividirlos por partes, poco a poco. Esto te ayudará a crear un hábito de aprendizaje más efectivo. Además, te sugiero que crees un repositorio en GitHub para todas tus prácticas, es decir un repositorio donde contengas todo tu proceso de aprendizaje. Esto no solo te ayudará a seguir practicando con Git y GitHub, sino que también te permitirá llevar un registro de tu progreso, ya si deseas crear un proyecto mas robusto puedes crear un repositorio individual para el.

Lamentablemente, no puedo sugerir un tutorial específico en inglés, ya que pertenezco a la comunidad hispana. Sin embargo, la clave es encontrar un tutorial que se adapte a tu ritmo y estilo de aprendizaje.

**Reply by Aimpact-naveen** [reply]
Thanks for your suggestion again Carlo, I will do exactly that sequence. Regards

**Comment by y0geshx** [comment]
I recommend, you to start with GitHub profile Readme 🙂

**Reply by Aimpact-naveen** [reply]
Hi Yogesh, thanks for the recommendation, Am looking at at it now, Regards
